src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -309,7 +323,8 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		for (const completion of completions) {
 			if (!completion.icon && completion.kind !== undefined) {
 				completion.icon = this._kindToIconMap.get(completion.kind);
-				completion.kindLabel = this._kindToTypeMap.get(completion.kind);
+				completion.kindLabel = this._kindToKindLabelMap.get(completion.kind);
+				completion.type = this._kindToTypeMap.get(completion.kind);
```

Comment:
How about doing the mapping of `completion.kind` to a string inside `_sendTelemetryInfo` since it's only needed for telemetry.